docs/docs/statistics/power-technical.mdx

```diff
@@ -0,0 +1,346 @@
+
+We use power analysis described above, except we substitute $\tilde{\sigma}^{2}$ for $\hat{\sigma}_{\Delta}^{2}$.
+
+## Bayesian engine details
+
+### Bayesian power
+
+For Bayesian power analysis, we let users specify the prior distribution of the treatment effect.
+We then estimate Bayesian power, which is the probability that the $(1 - \alpha)$ credible interval does not contain 0.
+
+We assume a conjugate normal-normal model, as follows:
+
+$$
+\begin{align*}
+\Delta &\stackrel{}{\sim}\mathcal{N}\left(\mu_{prior}, \sigma_{prior}^{2}\right)
+\\ \hat{\Delta}|\Delta &\stackrel{}{\sim}\mathcal{N}\left(\Delta, \hat{\sigma}_{\Delta}^{2}\right).
+\end{align*}
+$$
+
+In words, the model has two parts: 1) the normal prior for the treatment effect, which is specified by you; and 2) conditional upon the treatment effect, the estimated effect is normally distributed.
+The normal prior has several advantages, including: 1) bell-shaped distribution around the prior mean, so that extreme estimates will be shrunk more towards the prior than moderate estimates; 2) the ability to specify two moments, which is often the right amount of information for a prior; and 3) simplicity.  
+The conditional normality of the effect estimate is motivated by the central limit theorem.
+
+We use the normal distribution below to approximate the posterior:
+
+$$
+\begin{align*}
+\Delta|\hat{\Delta} &\stackrel{}{\sim}\mathcal{N}\left(\Omega^{-1}\omega, \Omega^{-1}\right)
+\\\Omega &= 1/\sigma_{prior}^{2} + 1/\hat{\sigma}_{\Delta}^{2}
+\\\omega &= \mu_{prior}/\sigma_{prior}^{2} + \hat{\Delta}/\hat{\sigma}_{\Delta}^{2}.
+\end{align*}
+$$
+
+This is an approximation to the posterior because $\Delta$ affects $\hat{\sigma}_{\Delta}^{2}$. We tested this approximation through extensive simulations, and found it had comparable coverage and mean squared error to a posterior distribution empirically sampled using Metropolis Hastings.
+
+We define rejection as the $100(1-\alpha)$\% confidence interval not containing zero.  
+For our posterior approximation, this occurs if the posterior mean for $\Delta|\hat{\Delta}$ (i.e., $\Omega^{-1}\omega$) divided by its posterior standard deviation$\left(\text{i.e.,  }\sqrt{\Omega^{-1}}\right)$ is beyond the the appropriate critical threshold $Z^{\star}$ (e.g., $\Phi^{-1}(0.975)$ for $\alpha=0.05$).
+
+Inside of a Bayesian framework, it can help to permit the case where the prior model is misspecified.  
+That is, the prior specified by the customer differs from the true prior that generates the treatment effect.  
+We permit misspecification of the prior for $\Delta$, as we assume that the true data generating process (DGP) is $\Delta \stackrel{}{\sim}\mathcal{N}\left(\mu_{\star}, \sigma_{\star}^{2}\right)$, while the specified DGP has $\Delta \stackrel{}{\sim}\mathcal{N}\left(\mu_{prior}, \sigma_{prior}^{2}\right)$.
```

Comment:
While "true DGP" makes sense, I feel that calling the two DGPs the true and the specified DGPs is a bit misleading. I imagine in most customer's eyes, they're thinking of the "true DGP" they write down in the modal as the desired effect size, while the prior is their belief about the effect size in general. They want to know the answer to the question: "Given my prior is X, what is the likelihood that if the true effect is actually Y, that I can reach statistical significance?"

So in that way, maybe we need to find another term besides "specified DGP". WDYT?

Reply:
I agree that the customer often wants to have answered, "Given my prior is X, what is the likelihood that if the true effect is actually Y, that I can reach statistical significance?"  I used your language in my doc at a later point to emphasize this.  

I like "specified" in that specifying a prior is more all-encompassing than the term "prior belief".  Sometimes when you specify a prior, it is just your belief.  But sometimes prior specification comes from a previous study or another data driven source.  So I like "specified".  But I'm open to other terms.